monkey/monkey_island.py

```diff
@@ -32,8 +31,7 @@
     # We need to initialize environment singleton before importing main,
     # because main imports modules from monkey_island/cc/models and models need a connection to the
     # mongodb. Mongodb connection parameters are initialized in environment singleton.
-    env_singleton.initialize_from_file(server_config_path)
 
-    from monkey_island.cc.main import main  # noqa: E402
+    from monkey_island.cc.server_setup import setup_island  # noqa: E402
```

Comment:
Do we still need to import this here? I think this can be moved to the top of the file now, right?